test/metabase/api/card_test.clj

```diff
@@ -733,7 +733,7 @@
     (testing "Fetch card with an emitter"
       (mt/with-temp* [Card [read-card {:name "Test Read Card"}]
                       Card [write-card {:is_write true :name "Test Write Card"}]
-                      CardEmitter [emitter {:action_id (db/select-field :action_id QueryAction :card_id (u/the-id write-card))
+                      CardEmitter [emitter {:action_id (u/the-id (db/select-one-field :action_id QueryAction :card_id (u/the-id write-card)))
```

Comment:
`select-field` returns a collection of values of that Field. `select-one-field` returns one specific value of that Field. Not sure how this ever actually worked with passing in a collection for `:action_id`